service/src/test/java/org/cbioportal/service/impl/MutationalSignatureServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package org.cbioportal.service.impl;
+
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import org.cbioportal.model.GeneticProfile;
+import org.cbioportal.model.MutationalSignature;
+import org.cbioportal.model.SNPCount;
+import org.cbioportal.model.Sample;
+import org.cbioportal.persistence.GeneticProfileRepository;
+import org.cbioportal.persistence.MutationalSignatureRepository;
+import org.cbioportal.persistence.SampleRepository;
+import org.cbioportal.service.MutationalSignatureService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+@RunWith(MockitoJUnitRunner.class)
+public class MutationalSignatureServiceImplTest {
+    @Test
+    public void noContextSignaturesTest() throws Exception {
+	    List<SNPCount> testSNPCounts = new LinkedList<>();
+	    testSNPCounts.add(new SNPCount("sample1", "A", "T", 15));
+	    testSNPCounts.add(new SNPCount("sample1", "C", "T", 5));
+	    testSNPCounts.add(new SNPCount("sample1", "G", "T", 20));
+	    testSNPCounts.add(new SNPCount("sample1", "C", "T", 40));
+	    testSNPCounts.add(new SNPCount("sample1", "T", "C", 1));
+	    
+	    List<String> sampleIds = new LinkedList<>();
+	    sampleIds.add("sample1");
+	    
+	    List<MutationalSignature> signatures = MutationalSignatureServiceImpl.MutationalSignatureFactory.NoContextSignatures(testSNPCounts, sampleIds);
```

Comment:
I was just making a comment to myself ... so I remember what the test function is testing (during a second review or readthrough of the pull request)